client/signup/jetpack-connect/plans.jsx

```diff
@@ -156,16 +85,10 @@ const Plans = React.createClass( {
 							steps={ 3 } />
 
 						<div id="plans" className="plans has-sidebar">
-							<PlanList
-								isInSignup={ true }
+							<PlansFeaturesMain
 								site={ selectedSite }
-								plans={ plans.get() }
-								sitePlans={ this.props.sitePlans }
-								cart={ this.props.cart }
-								showJetpackFreePlan={ true }
-								isSubmitting={ false }
-								onSelectPlan={ this.selectPlan }
-								onSelectFreeJetpackPlan={ this.selectFreeJetpackPlan }/>
+								intervalType="yearly"
+								hideFreePlan={ false } />
```

Comment:
This is probably redundant as `hideFreePlan` is set to `false` by default here: https://github.com/Automattic/wp-calypso/blob/78bb5f44accad894b8d1d3ec0e3ee11b5d94a5d7/client/my-sites/plans-features-main/index.jsx#L298-L298.